.gitignore

```diff
@@ -0,0 +1,2 @@
+.*.sw*
+.sw*
```

Comment:
I did this at work and a coworker said no text temp files in gitignore and instead they should be in our global gitignore that way all projects on your machine get to ignore swap files and tags and the individual project don't get cluttered with them.

http://stackoverflow.com/a/7335487/227176